### ISSUE TITLE
Author final submission page multiple keywords not working (accessibility)

### ISSUE THREAD

**Issue by jonibar** [psu-stewardship/etda#102]
On the author final submission page when using Jaws, it is not possible to enter multiple keywords.  Issues include:

Unable to create multiple keywords
Unable to tab out of  the field
Unable to backspace and delete keyword item
Although autocomplete is not configured for tag-it on ETD, Jaws announces autocomplete which is confusing

Tag-it javascript does not include accessibility tags.  Need to find a different utility to use or return to 'Additional Keyword' button.  Admin interface uses 'Additional Keyword' button so that admins can edit a keyword.  Tag-it allows a keyword to be deleted and re-entered but does not have ability to change the content.

**Comment by jonibar** [comment]
Looking at pivotal tracker, tag-it is configured to accept multiple keywords separated by commas.  This configuration allows multiple words to be accepted as one keyword (ie 'Penn State,' = one keyword).  Help text should be entered above the keyword field to explain this.   After an explanation has been entered, ask Michelle to test using commas to separate keywords.  If Jaws cannot accept the comma separated list, my suggestion is to use the method used on the admin edit submission pages.

**Comment by jonibar** [comment]
Tag-it examples are showing aria labels but I don't see them on our implementation of tag-it.
